WORKSPACE

```diff
@@ -91,3 +91,13 @@ npm_import(
     root_package = "",
     version = "6.32.19",
 )
+
+npm_translate_lock(
+    name = "npm_translate_package_lock",
+    npm_package_json = "//examples/npm_translate_package_lock:package.json",
```

Comment:
this is undesirable, we don't want to invalidate repo rules just because a non-dependency-related change was made to package.json - can we avoid reading it?